docs/installation_guide.md

```diff
@@ -366,109 +411,128 @@ DATABASES = {
 
 ## Database population
 
-Before running WireCloud, it is necessary to populate the database. This can be
-achieved by using this command:
+Before running WireCloud, it is necessary to populate the database. This can be achieved by using this command:
 
-    $ python manage.py migrate
+```bash
+python manage.py migrate
+```
 
-This command is used for migrating the database between different versions of
-WireCloud, but when executed the first time, it will serve for creating the
-initial database structure. Once create, we have to create a superuser to be
-able to login into WireCloud and to be able to perform administrative tasks from
-the web interface. Use the `createsuperuser` command for creating such a user.
-An example of the command output, where `user`/`password` are `admin`/`admin`,
-is the following:
+This command is used for migrating the database between different versions of WireCloud, but when executed the first
+time, it will serve for creating the initial database structure. Once create, we have to create a superuser to be able
+to login into WireCloud and to be able to perform administrative tasks from the web interface. Use the `createsuperuser`
+command for creating such a user. An example of the command output, where `user`/`password` are `admin`/`admin`, is the
+following:
 
+```bash
+python manage.py createsuperuser
+```
 
-    $ python manage.py createsuperuser
+```text
     Username (leave blank to use 'wirecloud'): admin
     Email address: admin@myemaildomain.com
     Password: ***** (admin)
     Password (again): ***** (admin)
     Superuser created successfully.
+```
 
-WireCloud 1.0 added some predefined dashboards, so you have to create them by
-running the following command:
+WireCloud 1.0 added some predefined dashboards, so you have to create them by running the following command:
 
+```bash
+python manage.py populate
 ```
-$ python manage.py populate
-```
-
 
 ## Search indexes configuration
 
-Wirecloud uses [Haystack](http://haystacksearch.org/) to handle the search
-indexes.
+Wirecloud uses [Haystack](http://haystacksearch.org/) to handle the search indexes.
 
-Currently, [Solr][], [ElasticSearch2][] and [Whoosh][] are supported. Whoosh is enabled by
-default.
+Currently, [Solr][], [ElasticSearch2][] and [Whoosh][] are supported. Whoosh is enabled by default.
 
-To modify the search engine configuration, it is necessary to modify the
-`HAYSTACK_CONNECTIONS` configuration setting in the instance `settings.py` file
-(e.g. `/opt/wirecloud_instance/wirecloud_instance/settings.py`).
+To modify the search engine configuration, it is necessary to modify the `HAYSTACK_CONNECTIONS` configuration setting in
+the instance `settings.py` file (e.g. `/opt/wirecloud_instance/wirecloud_instance/settings.py`).
 
-[Solr]: http://lucene.apache.org/solr/
-[ElasticSearch2]: https://www.elastic.co/products/elasticsearch
-[Whoosh]: https://whoosh.readthedocs.io/en/latest/
+[solr]: http://lucene.apache.org/solr/
```

Comment:
Although this is not breaking documentation, I'm curious, why to change them to low case if the references on the paragraphs maintains the capitalisation?

Reply:
Fixed fb9c400 - I've inlined the URLs - it is better markdown practice anyway.